main.py

```
@@ -334,6 +336,34 @@ def _is_blocked_by_provider_wake_prefix(self, event: AstrMessageEvent) -> bool:
+
+            chat_id = event.unified_msg_origin
+            astrbot_conf = self.context.get_config(chat_id)
+            provider_settings = astrbot_conf.get("provider_settings", {}) if astrbot_conf else {}
+            provider_wake_prefix = (provider_settings.get("wake_prefix", "") or "").strip()
+            if not provider_wake_prefix:
+                return False
+
+            message_outline = ""
+            try:
+                message_outline = (event.get_message_outline() or "").strip()
+            except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)